portfolio/src/main/webapp/index.html

```diff
@@ -7,10 +7,10 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload = "onloader()">
     <div id="content">
 
-      <h1>Hi! I'm Awad</h1>
+      <h1 id = "Introduction"></h1>
```

Comment:
There shouldn't be spaces around the equals sign here

Reply:
Didn't catch that thanks!